client/src/containers/Corruption/CorruptionView.tsx

```diff
@@ -57,18 +64,20 @@ export const CorruptionView: FC<CorruptionViewProps> = ({ corruption }) => {
       gap={4}
       mb={4}
     >
-      <GridItem w="100%" onClick={() => setIsScoreActive(false)}>
-        <Box mb={4} cursor={'pointer'}>
+      <GridItem w="100%">
+        <Box mb={4} cursor={'pointer'}  onClick={() => {setIsScoreActive(false); setSelectedId(CORRUPTION_INDEX_IDS["CORRUPTION_RATING"]); }}>
```

Comment:
How do you highligh the default card? By default, Score indicator is selected and specific chart is shown.